repo/const.go

```diff
@@ -5,6 +5,6 @@ const (
 	ConfigFile = "config"
 	SpecsFile  = "datastore_spec"
 
-	SupportedRepoVersion = 9
-	ToolVersion          = "0.1.1"
+	SupportedRepoVersion = 10
+	ToolVersion          = "0.1.2"
```

Comment:
@magik6k @Stebalien what's the deal with the tool version? I notice it hasn't been bumped in ages (nor released to dist.ipfs.io).

Any objection to bumping this to v0.5.0, tagging it and doing a dist.ipfs.io release? We'd then do a matching minor tool version bump + dist release every time we either make a meaningful change or bump the go-ipfs repo version (and a patch change if there's a bug fix).

Reply:
We just haven't bumped it in a while. It should match the tag.

Reply:
Done. Will do a tag + release